cinch/site.yml

```diff
@@ -35,6 +35,7 @@
   hosts: jenkins_master
   roles:
     - beaker-client
+    - nginx
```

Comment:
This should be guarded with a when: ansible_connection != 'docker' clause, as we don't want extra services running within a Docker context.